app/helpers/schema_helper.py

```diff
@@ -64,4 +64,10 @@ def is_goto_rule(rule):
 
     @staticmethod
     def is_goto_meta_rule(rule):
```

Comment:
Except it no longer is, the rule can contain multiple when's so it could be a meta and non-meta at the same time now...